GreedyCounter/Form1.cs

```diff
@@ -10,11 +11,29 @@ public partial class Form1 : Form
 
         OpenFileDialog dialog;
 
+        System.Windows.Forms.Timer timer;
+
+        string output;
+
+        //Initialize the concurrent dictionary;
+        ConcurrentDictionary<string, int> cDict = new ConcurrentDictionary<string, int>();
+
+        //Initialize a stringbuilder
+        StringBuilder sb = new StringBuilder();
+
         public Form1()
         {
             InitializeComponent();
+
+            timer = new System.Windows.Forms.Timer();
+            timer.Interval = 1000;
+            timer.Tick += new EventHandler(timer_Tick);
+            timer.Start();
```

Comment:
[Query] Should we start the timer before we've selected a file? Could there be potential issues with this?

Reply:
Honestly, I have no idea. But starting the timer to after the log file has been selected seems ok too.